src/constant.h

```diff
@@ -7,594 +7,598 @@
 // Global constants used by Moria
 // clang-format off
 
-#define MAX_UCHAR         255
-#define MAX_SHORT       32767   // maximum short/long signed ints
-#define MAX_LONG   0x7FFFFFFFL
+#pragma once
+
+#include "headers.h"
+
+constexpr int32_t MAX_UCHAR = std::numeric_limits<unsigned char>::max();
+constexpr int32_t MAX_SHORT = std::numeric_limits<short>::max(); // maximum short/int32_t signed int32_ts
```

Comment:
Search/replace has changed a bunch of comments too. Both `long` and `int` have been replaced with `int32_t` throughout the file.